tox.ini

```diff
@@ -1,13 +1,8 @@
-# Tox (https://tox.readthedocs.io/) is a tool for running tests
-# in multiple virtualenvs. This configuration file will run the
-# test suite on all supported python versions. To use it, "pip install tox"
-# and then run "tox" from this directory.
-
 [tox]
-envlist = py26, py27, py35, py36
+envlist = py27,py38
```

Comment:
why not 3.7 (the default for debian)

Reply:
I'm actually wondering whether this should be left open to `py2` and `py3` so that it can run against whatever python 2 and 3 versions the user has installed.

Reply:
Alright, done. Tox should be able to run with whatever versions are installed now.